vimrc

```diff
@@ -188,6 +188,9 @@ let g:vim_markdown_folding_disabled = 1
 let g:go_fmt_command = "goimports"
 let g:go_highlight_trailing_whitespace_error = 0
 
+let test#strategy = "vimux"
+let test#python#runner = 'nose'
```

Comment:
Is this a value that's easily configurable? We've been writing some Python lately, but all our testing uses `pytest`, and it would be great to be able to run tests with vimux.

Reply:
How are you currently running them? I configured this to match existing behavior. I presume it's easily configurable but I'm not sure exactly how.

Reply:
We're just running the `pytest` command manually. The problem we ran into when initially investigating this is that Vimux doesn't seem to support multiple possible test runners per programming language, and we don't want to remove support for nose entirely since I assume some teams still use it.

Reply:
At the very least, you can just manually run this in vim and it will switch the runner for you: 
```
:let test#python#runner='pytest'
```